content/pricing/_index.md

```diff
@@ -39,6 +39,7 @@ tiers:
                 - OIDC and team access tokens
                 - Webhooks
                 - Automatic secrets rotation
+                - 3000 free deployment minutes per month
```

Comment:
Why is this necessary?
It also breaks the symmetry of same number of lines per tier as well as alignment of support.

Reply:
I'd argue that the term "free" is either redundant or erroneous. On the Individual plan, it's redundant, on the Team plan, it's just included, not free, so you could shorten both to just:

- 500 deployment minutes/month
- 3000 deployment minutes/month (or 3k if you really want it short)